Procfile

```diff
@@ -1 +1 @@
-web: gunicorn --timeout=60 --workers=8 --threads=8 server:app.main
+web: python3 -m app.main
```

Comment:
Why don't we use gunicorn for this?  I looked at the startup.sh file so I know this change is correct

Reply:
sdx-downstream is a rabbit consumer it doesn't have a web component so it can't be started by gunicorn. I missed it on Andrew's previous commit so fixed it here.